topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+## Carpentries Staff Immediate Response Checklist
+
+When a Code of Conduct (CoC) violation is found, one course of action is that activities in Carpentries channels (in-person and virtual) are monitored. In other cases, the community member is asked to take a termed suspension. When this is the resolution of the CoC committee, the following boundaries are enforced. The checklist below was created for incidents that require an immediate response, i.e. the community member is removed from all Carpentries engagement.
+
+The intended outcomes for such a suspension are immediate, ongoing, and demonstrable changes in any areas of engagement for which the CoC deemed undesirable in the incident review. 
+
+### Online Communication and Communities
+Revoke access from the following communication channels and communities: 
+- [ ] Email lists on TopicBox 
+- [ ] Email lists on Google Groups
+  - [ ] Library Carpentry Governance Group
+  - [ ] Library Carpentry Maintainer Onboarding
+  - [ ] Code of Conduct Committee
+  - [ ] Carpentries Mentoring Groups 
+- [ ] Twitter - (@datacarpentry, @swcarpentry, @libcarpentry, @thecarpentries)
+- [ ] Blogs - (Data Carpentry, Software Carpentry, Library Carpentry, The Carpentries)
+- [ ] GitHub (carpentries, datacarpentry, swcarpentry, librarycarpentry, data-lessons, carpentrieslab)
+- [ ] Gitter - Library Carpentry Lobby
+- [ ] [Slack](https://swcarpentry.slack.com/messages)  
+- [ ] Facebook 
+- [ ] LinkedIn 
+
+Additionally, the community member will not be allowed to attend community discussions.
+
+### Teaching Workshops
+The suspended member will, for the duration of their suspension:
+- [ ] Be asked not to sign up for workshops on the Instructor Spreadsheet.
+- [ ] Not be able to participate as a helper at any workshops.
+- [ ] Be removed from Eventbrite login access.
+- [ ] Not be allowed to plan self-organized workshops.
+- [ ] Not be allowed to request or host centrally coordinated workshops.
+- [ ] Not be listed as an instructor on the “Instructors” webpage.
```

Comment:
This is definitively a breach of confidentiality. This should only be done in cases where a situation is resolved. However, the EC might of course decide something here unilateraly.

Reply:
As I've mentioned elsewhere, I would agree that doing this during all reviews would be overkill and a breach of privacy.  However, totally on the table for the consequences of a suspension.

Reply:
Updated intro paragraph also resolves this.